### ISSUE TITLE
Doc. proposal

### ISSUE THREAD

**Issue by vingerha** [rospogrigio/localtuya#779]
Hi, am enjoying the work you did and I have one recommendation to add to the documentation.
'Local' also can be interpreted so that the devices can work local, i.e. without internet which (at least in my case) they do not.
I suggest to add a small piece of text to possibly explain that vs. the 'normal' integration of tuya in HA

**Comment by CloCkWeRX** [comment]
Hi @vingerha - a really good way to do this is via the wiki, or a pull request to the readme.

Try https://github.com/rospogrigio/localtuya/edit/master/README.md or forking the wiki.

**Comment by vingerha** [comment]
yeah...well... I donot want to fork it for something this simple... a single line on the main README.md suffices. Anyhow...I know how it works ...just thinking of others that may like to know this
